src/webapp/api/index.js

```diff
@@ -131,10 +137,17 @@ export default class Api {
   }
 
   amountToRequest() {
-    if (this.feeCalculator && this.feeCalculator.targetLamportsPerSignature) {
-      return 10 * this.feeCalculator.targetLamportsPerSignature;
+    // Request enough to create 100 rent exempt message accounts, that should be plenty
+    if (this.feeCalculator && this.feeCalculator.lamportsPerSignature) {
+      return (
+        100 *
+          (this.feeCalculator.lamportsPerSignature +
+            this.minimumBalanceForRentExemption) -
+        this.feeCalculator.lamportsPerSignature
+      );
     }
-    return 1000000;
+    // Otherwise some large number
+    return 10000000 - 5000; // - default fee used to transfer
```

Comment:
Why subtract the fee here and above?

Reply:
Doing so makes the amount requested from the wallet work out of the box.  The wallet default airdrop amount does not subtract the fees.  It will hopefully reduce confusion.

Reply:
I'm still confused, what does this have to do with wallet airdrop?

Reply:
This is the amount that shows up as a default amount to transfer when the wallet ui opens.  If folks just airdrop themselves the default amount of tokens its nice to have the default amount to transfer also just work.  The alternative is to just make sure the default airdrop amount is more than enough to cover the default transfer amount plus the fees to do the transfer.

Reply:
Ah ok, yeah I think the alternative would be better but this is fine. Thanks for the explanation!